platforms/software/dependency-management/src/main/java/org/gradle/internal/component/external/model/maven/MavenDependencyMetadata.java

```diff
@@ -71,6 +71,11 @@ protected ModuleDependencyMetadata withRequested(ModuleComponentSelector newSele
         return new MavenDependencyMetadata(newDescriptor, getReason(), isEndorsingStrictVersions(), getArtifacts());
     }
 
+    @Override
+    protected ModuleDependencyMetadata withArtifacts(List<IvyArtifactName> artifacts) {
+        return new MavenDependencyMetadata(dependencyDescriptor, getReason(), isEndorsingStrictVersions(), artifacts);
```

Comment:
for consistency, should this be using `getArtifacts()` like the others?  If so... can you also fix the `artifacts` just below too?

Reply:
Not sure what you mean. `artifacts` are passed in the method and we are using those artifacts. Where would you call `getArtifacts`?

Reply:
bah, I didn't see the parameter.  nevermind